main.go

```diff
@@ -229,6 +239,10 @@ func (p Ploop) Mount(target string, options map[string]string) (*flexvolume.Resp
 	if options["kubernetes.io/readwrite"] == "ro" {
 		readonly = true
 	}
+	if options["secretFromSystem"] == "true" {
+		options["kubernetes.io/secret/clusterName"] = os.Getenv("clusterName")
+		options["kubernetes.io/secret/clusterPassword"] = os.Getenv("clusterPassword")
```

Comment:
I think we should use more unique names for these variables to be sure that we don't intersect with someone else.
For example, they can be kubeVstorageClusterName and kubeVstorageClusterPassword.

BTW: how are you going to set these variables?

Reply:
No problem...I'll rename those variables.
Currently we are setting these variables through the kubelet's environment using systemd
```
# cat /etc/systemd/system/kubelet.service.d/15-ploop.conf
[Service]
EnvironmentFile=/etc/sysconfig/ploop-flexvol
```
```
# cat /etc/sysconfig/ploop-flexvol
clusterName="base64encodedClusterName"
clusterPassword="base64encodedPassword"
workingDir="/vstorage"
```